LICENSE-3rdparty.csv

```diff
@@ -285,81 +286,66 @@ core,"github.com/gobwas/glob/syntax/ast",MIT
 core,"github.com/golang/glog",Apache-2.0
 core,"github.com/golang/groupcache/lru",Apache-2.0
 core,"github.com/golang/mock/gomock",Apache-2.0
-core,"github.com/golang/protobuf/descriptor",NewBSD
-core,"github.com/golang/protobuf/internal/gengogrpc",NewBSD
-core,"github.com/golang/protobuf/jsonpb",NewBSD
-core,"github.com/golang/protobuf/proto",NewBSD
-core,"github.com/golang/protobuf/protoc-gen-go",NewBSD
-core,"github.com/golang/protobuf/protoc-gen-go/descriptor",NewBSD
-core,"github.com/golang/protobuf/protoc-gen-go/generator",NewBSD
-core,"github.com/golang/protobuf/protoc-gen-go/generator/internal/remap",NewBSD
-core,"github.com/golang/protobuf/protoc-gen-go/plugin",NewBSD
-core,"github.com/golang/protobuf/ptypes",NewBSD
-core,"github.com/golang/protobuf/ptypes/any",NewBSD
-core,"github.com/golang/protobuf/ptypes/duration",NewBSD
-core,"github.com/golang/protobuf/ptypes/struct",NewBSD
-core,"github.com/golang/protobuf/ptypes/timestamp",NewBSD
-core,"github.com/golang/protobuf/ptypes/wrappers",NewBSD
-core,"github.com/golang/snappy",NewBSD
-core,"github.com/google/go-cmp/cmp",NewBSD
-core,"github.com/google/go-cmp/cmp/internal/diff",NewBSD
-core,"github.com/google/go-cmp/cmp/internal/flags",NewBSD
-core,"github.com/google/go-cmp/cmp/internal/function",NewBSD
-core,"github.com/google/go-cmp/cmp/internal/value",NewBSD
+core,"github.com/golang/protobuf/descriptor",BSD-3-Clause
+core,"github.com/golang/protobuf/internal/gengogrpc",BSD-3-Clause
+core,"github.com/golang/protobuf/jsonpb",BSD-3-Clause
+core,"github.com/golang/protobuf/proto",BSD-3-Clause
+core,"github.com/golang/protobuf/protoc-gen-go",BSD-3-Clause
+core,"github.com/golang/protobuf/protoc-gen-go/descriptor",BSD-3-Clause
+core,"github.com/golang/protobuf/protoc-gen-go/generator",BSD-3-Clause
+core,"github.com/golang/protobuf/protoc-gen-go/generator/internal/remap",BSD-3-Clause
+core,"github.com/golang/protobuf/protoc-gen-go/plugin",BSD-3-Clause
+core,"github.com/golang/protobuf/ptypes",BSD-3-Clause
+core,"github.com/golang/protobuf/ptypes/any",BSD-3-Clause
+core,"github.com/golang/protobuf/ptypes/duration",BSD-3-Clause
+core,"github.com/golang/protobuf/ptypes/struct",BSD-3-Clause
+core,"github.com/golang/protobuf/ptypes/timestamp",BSD-3-Clause
+core,"github.com/golang/protobuf/ptypes/wrappers",BSD-3-Clause
+core,"github.com/golang/snappy",BSD-3-Clause
+core,"github.com/google/go-cmp/cmp",BSD-3-Clause
+core,"github.com/google/go-cmp/cmp/internal/diff",BSD-3-Clause
+core,"github.com/google/go-cmp/cmp/internal/flags",BSD-3-Clause
+core,"github.com/google/go-cmp/cmp/internal/function",BSD-3-Clause
+core,"github.com/google/go-cmp/cmp/internal/value",BSD-3-Clause
 core,"github.com/google/gofuzz",Apache-2.0
 core,"github.com/google/gofuzz/bytesource",Apache-2.0
-core,"github.com/google/gopacket",NewBSD
-core,"github.com/google/gopacket/afpacket",NewBSD
-core,"github.com/google/gopacket/layers",NewBSD
-core,"github.com/google/pprof/driver",Apache-2.0
```

Comment:
I am guessing these went away because they are no longer used by `wwhrd`? It seems weird though 🤔